### PR TITLE
Add mobile navigation using hamburger menu

### PR DESCRIPTION
**Pull request by BranBayou**
- I created a mobile-hamburger-menu branch for this project.
- I included javascript linters for this branch mobile-hamburger-menu.
- I designed a mobile pop-up menu with the appropriate links.
- I and my coding partners wrote a javascript code to pop up the mobile menu when the user clicks the hamburger menu.
- I and my coding partners wrote a javascript code to close the mobile menu when the user clicks the x button.
- I added the links to the mobile menu lists to navigate the page accordingly.

### CODE REVIEW

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# INVALID ❌ ❌ ❌ 

Hi @BranBayou, 👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌
 - This is the first project using Javascript and the Javascript linters should be set up correctly. I see that you added the `.eslintrc.json` file, but the `linters.yml` file in the `.github/workflows` folder should be updated too. Please update that file following [these instructions](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

----
_Invalid Code Review Request does not count into the code reviews limit._

File: styles.css [hunk 1]
```diff
@@ -17,6 +17,30 @@
...
+    padding: 60px 20px;
+  }
+
+  .menu-li {
+    font-family: arial, sans-serif, 'inter';
+    font-size: 32px;
+    font-weight: 700;
+    padding: 10px 20px;
+  }
+
```
Comment (omar-muhamad) [inline styles.css]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#:~:text=that%20you%20chose.-,You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins),-using%20the%20templates), Please add the correct margin top, bottom, and left values to the mobile menu options to make its position match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| image | image  |

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi Team,

### Nice job, you are almost there.

cat-driving-serious

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please add your collaborators names and social links to the `Authors` section of the README file to make it more professional.

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by AnselemOdims — APPROVED** [review]
### Hi  @BranBayou 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**